src/Engine/ProtoAssociative/CodeGen.cs

```diff
@@ -3318,7 +3318,7 @@ private void EmitCallingForBaseConstructor(int thisClassIndex, ProtoCore.AST.Ass
                 int bidx = core.ClassTable.ClassNodes[globalClassIndex].Base;
                 if (bidx != Constants.kInvalidIndex )
                 {
-                    int cidx = core.ClassTable.ClassNodes[bidx].ProcTable.
+                    int pidx = core.ClassTable.ClassNodes[bidx].ProcTable.
```

Comment:
Ah, it was meant to be the constructor index, not class index, gotcha!

Reply:
pidx = procedureIndex 
so that we do not confuse it with class index